File: Python/09. Prime number.py
number = int(input("Enter a number:"))
result = True

if( number < 2 ):
    result = False
for i in range(2,number):
    if( number % i == 0 ):
        result = False

if(result==True):
    print(number, "is a Prime Number")
else:
    print(number,"is not a Prime Number")

File: Java/09. Prime number.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the number to check Prime:");
        int number = userInput.nextInt();
        boolean res = true;

        if ( number == 0 || number == 1 ) {
            res = false;
        }
        
        for( int i=2; i<number; i++ ) {
            if( number % i == 0 ) {
                res = false;
                break;
            }
        }
        
        if ( res == true ) {
            System.out.println(number + " is a Prime Number");
        } else {
            System.out.println(number + " is not Prime Number");
        }
        userInput.close();
    }
}
